File: app/src/main/java/com/karthik/splash/Models/PhotoDetail/Exif.java
package com.karthik.splash.Models.PhotoDetail;


import com.google.gson.annotations.SerializedName;


public class Exif{

	@SerializedName("exposure_time")
	private String exposureTime;

	@SerializedName("aperture")
	private String aperture;

	@SerializedName("focal_length")
	private String focalLength;

	@SerializedName("iso")
	private int iso;

	@SerializedName("model")
	private String model;

	@SerializedName("make")
	private String make;
}

File: app/src/main/java/com/karthik/splash/models/photodetail/Exif.kt
package com.karthik.splash.models.photodetail


import com.google.gson.annotations.SerializedName


data class Exif(@SerializedName("exposure_time")
                private val exposureTime: String? = null,
                @SerializedName("aperture")
                private val aperture: String? = null,
                @SerializedName("focal_length")
                private val focalLength: String? = null,
                @SerializedName("iso")
                private val iso: Int = 0,
                @SerializedName("model")
                private val model: String? = null,
                @SerializedName("make")
                private val make: String? = null)